File: cms_server/cms_server/Entities/ConsultMedProbEntity.cs
using System;

namespace cms_server.Entities
{
    public class ConsultMedProbEntity
    {
        public int? cr_med_prob_pk { get; set; }
        public string consult_req_pk { get; set; }
        public string med_prob_pk { get; set; }
        public string med_prob_desc { get; set; }
        public string med_prob_host { get; set; }
        public string is_active { get; set; }
        public DateTime? encoded_at { get; set; }
        public string encoder_pk { get; set; }
    }
}

File: cms_web/src/Services/Entities/ConsultMedProbEntity.ts
interface ConsultMedProbEntity {
  cr_med_prob_pk?: number;
  consult_req_pk?: string;
  med_prob_pk?: string;
  med_prob_desc?: string;
  med_prob_host?: string;
  is_active?: "y" | "n";
  encoded_at?: string | Date;
  encoder_pk?: string;
}

export interface ConsultMedProbTableModel {
  limit: number;
  count: number;
  begin: number;
  table: Array<ConsultMedProbEntity>;
}

export default ConsultMedProbEntity;
